### ISSUE TITLE
asciiquarium segfaults 

### ISSUE THREAD

**Issue by mkschulze** [Homebrew/homebrew-core#73306]
### `brew gist-logs <formula>` link OR `brew config` AND `brew doctor` output

```bash
abc ~$ brew config
HOMEBREW_VERSION: 3.0.7
ORIGIN: https://github.com/Homebrew/brew
HEAD: 59ada80ca740e72d29e625079395ee91cae4b3cf
Last commit: 8 hours ago
Core tap ORIGIN: https://github.com/Homebrew/homebrew-core
Core tap HEAD: cc9c5a2d3f1fd3952aec6f06928b584091143878
Core tap last commit: 15 minutes ago
Core tap branch: master
HOMEBREW_PREFIX: /usr/local
HOMEBREW_CASK_OPTS: []
HOMEBREW_EDITOR: code
HOMEBREW_MAKE_JOBS: 16
Homebrew Ruby: 2.6.3 => /System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/bin/ruby
CPU: 16-core 64-bit dunno
Clang: 12.0 build 1200
Git: 2.24.3 => /Applications/Xcode.app/Contents/Developer/usr/bin/git
Curl: 7.64.1 => /usr/bin/curl
macOS: 11.2.3-x86_64
CLT: N/A
Xcode: 12.4
```

### 

- [X] The `brew doctor` above contains no "Warning" lines.

### What were you trying to do (and why)?

I would like to start asciiquarium on macOS Big Sur 

### What happened (include all command output)?

```bash
abc ~$ brew install asciiquarium
Updating Homebrew...
==> Auto-updated Homebrew!
Updated 1 tap (homebrew/cask).
==> Updated Casks
Updated 1 cask.

==> Downloading https://homebrew.bintray.com/bottles/asciiquarium-1.1_1.catalina.bottle.tar.gz
Already downloaded: /Users/ulzee/Library/Caches/Homebrew/downloads/f24bfbe673b9bf8adb176938925346bdbaa66f68984c649b98e9783e8fbf96a4--asciiquarium-1.1_1.catalina.bottle.tar.gz
==> Pouring asciiquarium-1.1_1.catalina.bottle.tar.gz
🍺  /usr/local/Cellar/asciiquarium/1.1_1: 17 files, 373.1KB
---  00:31:22
abc ~$ asciiquarium
Segmentation fault: 11
```

### What did you expect to happen?

It should start up

### Step-by-step reproduction instructions (by running `brew` commands)

```bash
abc ~$ brew install asciiquarium
Updating Homebrew...
==> Auto-updated Homebrew!
Updated 1 tap (homebrew/cask).
==> Updated Casks
Updated 1 cask.

==> Downloading https://homebrew.bintray.com/bottles/asciiquarium-1.1_1.catalina.bottle.tar.gz
Already downloaded: /Users/ulzee/Library/Caches/Homebrew/downloads/f24bfbe673b9bf8adb176938925346bdbaa66f68984c649b98e9783e8fbf96a4--asciiquarium-1.1_1.catalina.bottle.tar.gz
==> Pouring asciiquarium-1.1_1.catalina.bottle.tar.gz
🍺  /usr/local/Cellar/asciiquarium/1.1_1: 17 files, 373.1KB
---  00:31:22
abc ~$ asciiquarium
Segmentation fault: 11
```

probably related: https://github.com/Homebrew/homebrew-core/issues/4936

**Comment by carlocab** [comment]
Related: #70297

**Comment by mkschulze** [comment]
also here: https://github.com/Homebrew/homebrew-core/pull/66485

**Comment by mkschulze** [comment]
Don't let it go stale, we NEED the Quarium :D

**Comment by iMichka** [comment]
Should be fixed with #70297.

Have fun with the fishes :D
